builtin/providers/template/resource.go

```diff
@@ -87,6 +94,8 @@ func Read(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
+type TemplateRenderError error
```

Comment:
Good call. I was just so excited that this was my first time using error types in Go that I wanted the _whole world_ to see my error type! :grin: